### DISCUSSION TITLE
Make Image for initContainer configurable

### DISCUSSION THREAD

**Discussion by Mezzenine** [Opster/opensearch-k8s-operator#206] [Q&A]
https://github.com/Opster/opensearch-k8s-operator/blob/66df6da40424a4a0d957b196656eebaf2cf9e1a2/opensearch-operator/pkg/builders/cluster.go#L280

Hi,
is there a possibility to configure the image registry for the initContainer image?

We only allow image pulls from our own registry.  Image pulls from public.ecr.aws is not allowed, regarding the company policy.

**Comment by swoehrl-mw** [comment]
Hi @Mezzenine. This is currently not possible, the busybox image is hardcoded. There was some discussion in #127, maybe @idanl21 can give an update on what is planned there.
